lib/OpenQA/Schema/Result/JobGroups.pm

```diff
@@ -107,4 +107,14 @@ around 'default_priority' => sub {
     return $self->get_column('default_priority') // ($self->parent ? $self->parent->default_priority : OpenQA::Schema::JobGroupDefaults::PRIORITY);
 };
 
+sub rendered_description {
+    my ($self) = @_;
+
+    if ($self->description) {
+        my $m = CommentsMarkdownParser->new;
+        return Mojo::ByteStream->new($m->markdown($self->description));
+    }
+    return 0;
```

Comment:
so just add a "return;" at the end of every function if it's not returning something else.